### PR TITLE
Task #118 done

### LINKED ISSUE #118: (SP: 1) Write unit test for "GeneralInfoStep" container
### Container unit test
Unit test for "**AddPhotoStep**" container
Scenaries descriptions:
- [ ] Container should rendered
- [ ] Check if the buttons passed in props is in the document

[Link to component](https://github.com/Space2Study-Project/Space2Study-FrontEnd/blob/develop/src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by AndriiNasadiuk**
close #118 

Знімок екрана 2023-11-30 о 22 02 31

Знімок екрана 2023-11-30 о 22 02 20

### CODE REVIEW

**Review by LeonidShamarin — APPROVED** [review]

**Review by marianamartsy — APPROVED** [review]
Please add a topic if this task is related to PopupDialog or GeneralInfoStep ;-)

GeneralInfoStep approved

**Review by ArtemHolikov — APPROVED** [review]
Seems good, but do we have another way to write test with delay in PopupDialog component?

**Review by Mygal-Alex — APPROVED** [review]

File: tests/unit/containers/tutor-home-page/general-info-step/GeneralInfoStep.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/containers/tutor-home-page/general-info-step/GeneralInfoStep.spec.jsx:1]: We need to delete this file from this PR?

File: tests/unit/components/popup-dialog/PopupDialog.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  })
+
+  it('should imitate click with delay and popup should be closed', async () => {
+    const popup = screen.getByTestId('popup')
+    fireEvent.click(popup)
+    fireEvent.click(popup)
+
+    await vi.fn(() => {
+      expect(closeModalAfterDelay).toHaveBeenCalledTimes(2)
+    })
```
Comment (vlad-khrychov) [inline tests/unit/components/popup-dialog/PopupDialog.spec.jsx]: it is doesn`t work

File: tests/unit/components/popup-dialog/PopupDialog.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    expect(content).toBeInTheDocument()
+  })
+
+  it('should imitate click on the close button and popup should be closed', async () => {
+    const closeButton = screen.getByTestId('popup').querySelector('button')
+    fireEvent.click(closeButton)
+
+    vi.fn(() => {
+      expect(closeModal).toHaveBeenCalledTimes(1)
+    })
```
Comment (vlad-khrychov) [inline tests/unit/components/popup-dialog/PopupDialog.spec.jsx]: Where you found this approach to run test? `vi.fn()` - https://vitest.dev/api/vi.html#vi-fn

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
Not correct way to run tests whith mocks

**Review by vlad-khrychov — APPROVED** [review]
Well done

**Pull request merged**